assets/src/components/form/inputs/base/input-help-text.js

```diff
@@ -8,7 +8,10 @@ import PropTypes from 'prop-types';
  */
 const InputHelpText = ( { helpTextID, helpText } ) => {
 	return helpText ? (
-		<p id={ helpTextID } className="espresso-input-help-text" >
+		<p
+			id={ helpTextID }
+			className={ 'espresso-input-help-text ee-focus-priority-6' }
```

Comment:
`ee-focus-priority-6` - magic class name :)

https://media.giphy.com/media/12NUbkX6p4xOO4/giphy.gif

Reply:
... not sure whether you are implying that is good ro bad { : \

here's an explanation of what all is rolled into the "focus priority" classes:
https://brent.eventespresso.com/wp-content/plugins/event-espresso-core/assets/src/components/ui/styles/themes/default/demo/index.html#font-sizes

Reply:
Makes sense, made a search through files and stumbled across: `--ee-font-size-extreme`, `--ee-font-size-bigger` and got distracted :)

Reply:
Saw this:
```
@media screen and ( max-width: 782px ) {
 .
.
.
    .ee-focus-priority-6 {
        font-size: var(--ee-font-size-smaller);
    }
.
.
.
}

```
and didn't understood at first, but after your comment got it